src/python/WMCore/Services/ReqMgr/ReqMgr.py

```diff
@@ -206,15 +206,15 @@ def updateRequestProperty(self, request, propDict):
         propDict["RequestName"] = request
         return self["requests"].put('request/%s' % request, propDict)[0]['result']
 
-    def getAbortedAndForceCompleteRequestsFromMemoryCache(self):
+    def getAbortedAndForceCompleteRequestsFromMemoryCache(self, expire=10):
         """
         _getAbortedAndForceCompleteRequestsFromMemoryCache_
         """
         # imports here to avoid the dependency not using this function
         from WMCore.Cache.GenericDataCache import MemoryCacheStruct
 
         maskStates = ["aborted", "aborted-completed", "force-complete"]
-        return MemoryCacheStruct(expire=0, func=self.getRequestByStatus, initCacheValue=[],
+        return MemoryCacheStruct(expire=expire, func=self.getRequestByStatus, initCacheValue=[],
```

Comment:
10 seconds should be good. I'm just afraid the cache might be stale for some workflows and, for instance, a request transition can be not allowed and fail. Or it can't happen?

Reply:
You are right. I think it it safer to set expire=0 as default